src/api/AppApi.js

```diff
@@ -225,3 +226,499 @@ export function installApp(appId :UUID, organizationId :UUID, prefix :string) :P
       return Promise.reject(error);
     });
 }
+
+/**
+  * `POST /app`
+  *
+  * Creates a new app.
+  *
+  * @static
+  * @memberof lattice.AppApi
+  * @param {Object} app
+  * @return {Promise} - a Promise that will resolve without a value after creating an app
+  *
+  * @example
+  * AppApi.createApp({
+  *   "name": "myapp",
+  *   "title": "My App",
+  *   "description": "This is my app.",
+  *   "appTypeIds": ["ec6865e6-e60e-424b-a071-6a9c1603d735"],
+  *   "url": "https://openlattice.com/my_app"
+  * });
+  */
+export function createApp(app :Object) :Promise<*> {
+
+  return getApiAxiosInstance(APP_API)
+    .post('/', app)
+    .then(axiosResponse => axiosResponse.data)
+    .catch((error :Error) => {
+      LOG.error(error);
+      return Promise.reject(error);
+    });
+}
+
+/**
+  * `POST /app/type`
+  *
+  * Creates a new app type.
+  *
+  * @static
+  * @memberof lattice.AppApi
+  * @param {Object} appType
+  * @return {Promise} - a Promise that will resolve without a value after creating an app type
+  *
+  * @example
+  * AppApi.createAppType({
+  *   "type": {
+  *     "namespace": "sample",
+  *     "name": "apptype"
+  *   },
+  *   "title": "Sample App Type",
+  *   "description": "This is a sample app type.",
+  *   "entityTypeId": "ec6865e6-e60e-424b-a071-6a9c1603d735"
+  * });
+  */
+export function createAppType(appType :Object) :Promise<*> {
+
+  return getApiAxiosInstance(APP_API)
+    .post(`/${TYPE_PATH}`, appType)
+    .then(axiosResponse => axiosResponse.data)
+    .catch((error :Error) => {
+      LOG.error(error);
+      return Promise.reject(error);
+    });
+}
+
+/**
+  * `GET /app/type/{appId}`
+  *
+  * Loads app type with provided id
+  *
+  * @static
+  * @memberof lattice.AppApi
+  * @param {UUID} appTypeId
+  * @return {Promise<Object>} - a Promise that will resolve with the details of an app type
+  *
+  * @example
+  * AppApi.getAppType("0c8be4b7-0bd5-4dd1-a623-da78871c9d0e");
+  */
+
+export function getAppType(appTypeId :UUID) :Promise<*> {
+
+  let errorMsg = '';
+
+  if (!isValidUuid(appTypeId)) {
+    errorMsg = 'invalid parameter: appTypeId must be a valid UUID';
+    LOG.error(errorMsg, appTypeId);
+    return Promise.reject(errorMsg);
+  }
+
+  return getApiAxiosInstance(APP_API)
+    .get(`/${TYPE_PATH}/${appTypeId}`)
+    .then(axiosResponse => axiosResponse.data)
+    .catch((error :Error) => {
+      LOG.error(error);
+      return Promise.reject(error);
+    });
+}
+
+/**
+  * `GET /app/type/lookup/{namespace}/{name}`
+  *
+  * Loads app type with provided namespace and name
+  *
+  * @static
+  * @memberof lattice.AppApi
+  * @param {string} namespace
+  * @param {string} name
+  * @return {Promise<Object>} - a Promise that will resolve with the details of an app type
+  *
+  * @example
+  * AppApi.getAppTypeByFqn("sample", "apptype");
+  */
+
+export function getAppTypeByFqn(namespace :string, name :string) :Promise<*> {
```

Comment:
I recommend using `FullyQualifiedName` instead. For reference, check out `EntityDataModelApi.getEntityTypeId()` and `EntityDataModelApi.getPropertyTypeId()`.